ui/js/directives/treeherder/clonejobs.js

```diff
@@ -830,6 +818,9 @@ treeherder.directive('thCloneJobs', [
 
             $rootScope.$on(
                 thEvents.jobsLoaded, function(ev, platformData){
+                    platformData.forEach(function(value) {
+                        addAdditionalJobParameters(value.jobGroups);
+                    });
```

Comment:
I think it is ok to put this stuff in updateJobs (at the very beginning), that will reduce a bit of the duplication here